server/webapp/WEB-INF/applicationContext-acegi-security.xml

```diff
@@ -261,7 +261,7 @@
                 /admin/package_repositories/**=ROLE_SUPERVISOR,ROLE_GROUP_SUPERVISOR
                 /admin/package_definitions/**=ROLE_SUPERVISOR,ROLE_GROUP_SUPERVISOR
                 /api/admin/pipelines=ROLE_SUPERVISOR,ROLE_GROUP_SUPERVISOR
-                /api/admin/pipelines/*=ROLE_SUPERVISOR,ROLE_GROUP_SUPERVISOR
+                /api/admin/pipelines/*=ROLE_USER
```

Comment:
Thats incorrect. This will allow all users to be able to access the pipeline config. It needs to be `ROLE_SUPERVISOR,ROLE_GROUP_SUPERVISOR`